.eslintignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
+node_modules/
+/.cache
+/build
+/server-build
+/public/build
+/coverage
+.build
+.cache
+package-lock.json
+pnpm-lock.yaml
+result
+.logs
+.pnpm-workspace.yaml
+*.yaml
+*.yml
+dist
+.dist
+**/dist/**
+dist
+logs
+*.toml
+*.md
```

Comment:
I don't think there's a need for this? We don't have toml files, nor server/build or /public/build, package-lock.json etc. And the linter only runs on .ts and .cjs files.

Was there a problem somewhere?

Reply:
I removed this file. I think it should be added in a separate pr with dist, yml/yaml and md. Without .eslintignore the linter lints files in dist which are supposed to be ignored.

Reply:
ok thanks, I'll look into it